release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -171,6 +171,11 @@ The `matchLabelKeys` parameter for configuring pod topology spread constraints i
 
 For more information, see xref:../nodes/scheduling/nodes-scheduler-pod-topology-spread-constraints.adoc#nodes-scheduler-pod-topology-spread-constraints[Controlling pod placement by using pod topology spread constraints].
 
+[id="ocp-4-14-MaxUnavailableStatefulSet"]
+==== MaxUnavailableStatefulSet enabled
+
+With this release, the `MaxUnavailableStatefulSet` is enabled to allow users to define the maximum number of StatefulSet pods that can be unavailable during updates. This reduces application downtown when upgrading.  
```

Comment:
Since "StatefulSet" is a specific type of pod, would it be in backticks? We do this for similar pods, eg `State` pod, `thanos-querier` pod etc. 

```suggestion
With this release, the `MaxUnavailableStatefulSet` featureset configuration parameter is enabled by default. This allows users to define the maximum number of `StatefulSet` pods that can be unavailable during updates, and reduces application downtown when upgrading.  
```
Is it a configuration parameter? I think it is, based on what I can find in the docs. Either way, we'd want to put _some_ noun phrase after MaxUnavailableStatefulSet. For example: ". . . the `MaxUnavailableStatefulSet` ____(noun/phrase)____ is enabled...

Some guidance: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

Reply:
I think you are right, thanks!

Reply:
@cbippley also i think downtown should be replaced by downtime

Reply:
Yeah, that makes sense. Whoops